db/seeds.rb

```diff
@@ -1,98 +1,7 @@
-city = %w[Poquott
-Comstock
-Northwest
-Bisbee
-Willcox
-Preston
-Nitro
-Sarcoxie
-Wattsburg
-Deep Water
-Saegertown
-Boston
-Antigo
-Frierson
-Venersborg
-Sunview
-Prestonsburg
-Menands
-Ettrick
-NYC
-Waban
-Worcester
-Purvis]
-
-name = [
-  "Molly Ward Gardens",
-  "Cambridge House",
-  "Georgian",
-  "Stonehouse Restaurant",
-  "Cafe Song Inc",
-  "Seasons Bistro & Grille",
-  "Zipper Factory Tavern",
-  "Bread's Arisin'",
-  "Chai Peking Chinese Restaurant",
-  "Heartline Cafe",
-  "Grants Restaurant & Bar",
-  "Pistone's Italian Inn",
-  "Pentimento",
-  "Paddy O'Brian's",
-  "Paulette's",
-  "Appalachia American Grill",
-  "Clery's Bar & Restaurant",
-  "Bistros Gourmet Deli & Pizza",
-  "Mac Gregor's Grill & Tap Room",
-  "J P China",
-  "Nick N Willy's Pizza",
-  "Joe Morley's Smoked Beef",
-  "Mr Jim's Pizza",
-  "Colonial Steak House",
-  "Harbor Lights",
-  "Patio Restaurants",
-  "PO-PO Family Restaurant",
-  "Coogans",
-  "Monticello Restaurant",
-  "Cozymel's Mexican Grill",
-  "Laudisio Ristorante Italiano",
-  "Frontroom Pizza",
-  "City Dock Cafe",
-  "Food 101 At Highlands Llc",
-  "Shiki Japanese Restaurant",
-  "World Famous Heartland Grill",
-  "Taco John's",
-  "Pub House",
-  "Toozypatza Pizza",
-  "Alley Cantina",
-  "Tony Z's Italian Restaurante",
-  "Franklin's Tower",
-  "LA Cocina Mexicana",
-  "Hickory Tavern",
-  "Crescent City",
-  "Raoul's Velvet Room",
-  "Wahpeton Deli",
-  "Thai Pilin Restaurant",
-  "Lolita Restaurant",
-  "Old Angler's Inn"
-]
-
-street_address = ['Broad Street',
-    'Lake Street',
-    'Hillside Drive',
-    'Route 202',
-    'Wall Street',
-    '3rd Street',
-    'Oxford Court',
-    '10th Street',
-    'Laurel Street',
-    'Olive Street',
-    '4th Avenue',
-    'Church Street South',
-    'Route 32',
-    'Sunset Drive',
-    'Lincoln Avenue',
-    'State Street',
-    'Hawthorne Lane',
-    'Locust Lane']
+require 'yelp'
+require 'pry'
+require 'json'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.